### ISSUE TITLE
Cannot copy any code on windows

### ISSUE THREAD

**Issue by numb92** [octref/polacode#28]
Every type of code I copy, I get the message: "Pasted content is invalid. Only copy from VS Code and check if your shortcuts for copy/paste have conflicts."

This happens with valid HTML and Javascript on VSCode for windows. 

Example of HTML code which is rejected:

`<table class="table table-bordered table-hover">
                <thead>
                    <tr>
                        <td>Frequency</td>
                        <td>When</td>
                    </tr>
                </thead>
                <tbody id="distributionTable">
                </tbody>
            </table>`

**Comment by sosoriov** [comment]
I got the same error message

this is mine:

```
    <v-loading loader='layerToServer'
               message='...'>
      <template slot='spinner'>
        <div class="loader3">
          <div class=" sk-folding-cube">
            <div class="sk-cube1 sk-cube"></div>
            <div class="sk-cube2 sk-cube"></div>
            <div class="sk-cube4 sk-cube"></div>
            <div class="sk-cube3 sk-cube"></div>
          </div>
        </div>
      </template>
    </v-loading>
```

**Comment by ueeieiie** [comment]
Same here

**Comment by amojicamu** [comment]
Apparently is not supported in Windows? I didn't see a OSX requirement listed.

Here's mine

`{
    "loginInfo": {
        "passwordSupported": true,
        "autoProvisioned": false
    }
}`

**Comment by cfjedimaster** [comment]
Polacode was working for me last week, but I'm now getting this as well. This is with the latest Insiders version of Code.

**Comment by octref** [comment]
Can you try a fewer things?

First, instead of using Copy, run the command “Copy With Syntax Highlight” to see if it works. 

Second, you can use https://pnp.gallery.vsassets.io/_apis/public/gallery/publisher/pnp/extension/polacode/0.1.2/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage to download 0.1.2 as a temp workaround.

Finally, if you are into fixing it, you can try opening Polacode, run Developer: Webview Tools and debug on innerHTML here: https://github.com/octref/polacode/blob/master/src/webview/index.js#L99

**Comment by octref** [comment]
I’m out today and can fix this later today.

**Comment by ajitid** [comment]
Checked and found this error occurs in both Ubuntu and Windows

**Comment by gnapse** [comment]
I'm having this issue too, in macOS, right off the bat after installing the extension.

**Comment by octref** [comment]
Try upgrade to 0.2.1.

**Comment by numb92** [comment]
I still get the same message. Sample code copied:

`function loadOverlay()
{
    $.ajax({
        url : "../overlay.html",
        success : function(result){
            displayOverlay(result);
        }
    });
}`

<div class="pageControls">
        <i class="fas fa-chevron-left ControlPrev" onclick="prevPage()"></i>
        <i class="fas fa-eraser clear" onclick="clearSelections()"></i>
        <span id="pageNumberDisplay"></span>
        <i class="fas fa-chevron-right ControlNext" onclick="nextPage()"></i>
    </div>

**Comment by octref** [comment]
0.2.2 should fix it and also a Windows padding issue (#7).
